### ISSUE TITLE
Tracing->format_timestamp causes intermittent errors due to incorrect handling of PHP microtime()

### ISSUE THREAD

**Issue by ravewebdev** [wp-graphql/wp-graphql#1565]
### Expected behavior ###

`$timestamp->format( 'Y-m-d\TH:i:s.uP' )` returns a properly formatted timestamp string.

### Actual behavior ###

`$timestamp->format( 'Y-m-d\TH:i:s.uP' )` throws an error, breaking our builds and deployments.

The following error is occurring intermittently (not always, not predictably, but often enough to be a problem) on our local and Vercel builds using WPGraphQL on a Jamstack headless WP build:

```
Uncaught Error: Call to a member function format() on bool in /code/wp-content/plugins/wp-graphql/src/Utils/Tracing.php:232 Stack trace: #0 /code/wp-content/plugins/wp-graphql/src/Utils/Tracing.php(113): WPGraphQL\Utils\Tracing->format_timestamp(1605027283)
```

After digging into the stack trace, it appears timestamp handling in `Tracing` is not taking into account that `microtime( true )` does not always return a float. [It can occasionally return an integer instead](https://www.php.net/manual/en/function.microtime.php#124984), which then breaks the timestamp formatting.

From `Tracing`:

```
$this->request_start_microtime = microtime( true );
$this->request_start_timestamp = $this->format_timestamp( $this->request_start_microtime );
...
public function format_timestamp( $time ) {
	$timestamp = \DateTime::createFromFormat( 'U.u', $time );

	return $timestamp->format( 'Y-m-d\TH:i:s.uP' );
}
```

When `$time` is an integer instead of a float, the `\DateTime::createFromFormat( 'U.u', $time )` line will return `false` instead of a DateTime object, so the following `$timestamp->format( 'Y-m-d\TH:i:s.uP' )` line will throw an error. The suggested fix from the php.net comment linked above is to force the result from `microtime( true )` to be a float.

We're running WPGraphQL version: 0.15.0

**Comment by jasonbahl** [comment]
@ravewebdev thanks for the detailed issue! I'll look at getting this addressed shortly, but if happy to accept a PR if you beat me to it!

**Comment by jasonbahl** [comment]
@ravewebdev I believe I resolved this as part of #1570. I implemented the solution suggested in the link you provided.
